packages/shared/ReactSymbols.js

```diff
@@ -24,8 +24,8 @@ export const REACT_STRICT_MODE_TYPE = hasSymbol
   ? Symbol.for('react.strict_mode')
   : 0xeacc;
 export const REACT_PROFILER_TYPE = hasSymbol
-  ? Symbol.for('react.profile_root')
-  : 0xeacc;
+  ? Symbol.for('react.profiler')
```

Comment:
This change isn't necessary. I just figured I'd match them up while I'm in here.